documentation/Lotus release flow

```diff
@@ -0,0 +1,138 @@
+
+# `lotus` Release Flow
+
+## Table of Contents
```

Comment:
Should we use a tool to generate this to ensure there are no mistakes: https://stackoverflow.com/questions/18244417/how-do-i-create-some-kind-of-table-of-content-in-github-wiki ?

Also, TOC will be automatically rendered in menu by GitHub: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/